test/external/imports/js-cjs/package.json

```diff
@@ -9,7 +9,7 @@
 	},
 	"dependencies": {
 		"express": "^4.17.2",
-		"express-rate-limit": "file:../../../../express-rate-limit-6.0.3.tgz"
+		"express-rate-limit": "file:../../../../express-rate-limit-6.0.4.tgz"
```

Comment:
Is this going to work once I bump it to 6.0.5 for this next release?

Reply:
Looking through the history, it looks like no, although I'm not sure I understand why...

Reply:
Hmmm this has started to bug me too :/

> Looking through the history, it looks like no, although I'm not sure I understand why...

It runs `npm install ../../../express-rate-limit-*.tgz` every time the test runs on CI/locally, so the version number is bumped - but not committed to git. When we run it locally though, the version number is bumped and if we do a `git add .`, it commits that change too.

I think we can link the package to the folder instead of a .tgz - that should remove this version bump thingie.

Reply:
Releasing 6.0.5 first, then will try out this fix

Reply:
Why can't we just specify the version and use the package from npm here? Just to know.

Reply:
Because we want to run the tests with the changes made on a branch on the repo before publishing to npm - there's no point in testing after publishing the module 😅

Reply:
Opened #283 for this

Reply:
I think we can create a GitHub Action that runs the test on that branch (any new branch), and if tests dont pass, publish wont happen.

Reply:
That is how it works currently - https://github.com/nfriedly/express-rate-limit/blob/master/.github/workflows/ci.yaml